src/server/game/Spells/SpellMgr.cpp

```diff
@@ -4948,6 +4948,13 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->RangeEntry = sSpellRangeStore.LookupEntry(5); // 40yd
     });
 
+    // The Smallest Creatures
+    ApplySpellFix({ 38544 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->_GetEffect(EFFECT_0).MiscValueB = 65;
+        spellInfo->_GetEffect(EFFECT_1).ApplyAuraName = SPELL_AURA_DUMMY;
```

Comment:
This is a hack. Player should be invisible while controlling the marmot

Reply:
works well even though it is a hack

Reply:
This is still a hack. Changing the aura type prevents the player from being invisible (blizzlike)

Reply:
I was surprised by this, I didn't imagine that the invisible effect would be impaired. However, the normal effect ( without the modification) causes skill problems. The control does not work as expected. As a result, the mission cannot be completed

Reply:
anyone can unhack this properly? this quest is needed to get horde outland quest achievment.

Reply:
@meji46 i made a recent change. You can test it again

Reply:
But the hack is still there and `GetCaster()->SetVisible()` isn't a valid workaround

Reply:
sooo, anyone can fix this hack properly?